typings/index.d.ts

```diff
@@ -114,6 +114,7 @@ declare module 'discord.js' {
     public readonly client: Client;
     public toJSON(...props: { [key: string]: boolean | string }[]): object;
     public valueOf(): string;
+    public static isValidID(string: string): string is Snowflake;
```

Comment:
same than https://github.com/discordjs/discord.js/pull/4915#discussion_r552859828.
```suggestion
    public static isValidID(id: string): string is Snowflake;
```

Reply:
ditto: https://github.com/discordjs/discord.js/pull/4915#discussion_r552961662